src/otx/core/model/entity/base.py

```diff
@@ -494,7 +494,24 @@ def _read_ptq_config_from_ir(self, ov_model: Model) -> dict[str, Any]:
 
         return argparser.instantiate_classes(initial_ptq_config).as_dict()
 
+    def _reset_prediction_layer(self, num_classes: int) -> None:
+        return
+
     @property
     def model_adapter_parameters(self) -> dict:
         """Model parameters for export."""
         return {}
+
```

Comment:
I just wonder why OVModel doesn't allow to update the num_classes, could please you explain? 
@kprokofi